src/components/Settings/SettingsTab.tsx

```diff
@@ -127,7 +127,8 @@ export const SettingsTab = ({
         <SettingRow
           icon={'clock'}
           name={`Make Room Permanent`}
-          description="Standard rooms are deleted after one day of inactivity. Permanent rooms aren't deleted and can have passwords/custom URLs. Free users can only have one permanent room at a time."
+          description="Standard rooms are deleted after one day of inactivity. Permanent rooms aren't deleted and can have passwords/custom URLs.
+          Free users can only have one permanent room at a time. Permanent rooms of subscribers have a greater room capacity."
```

Comment:
I'd move "Permanent rooms of subscribers have a greater room capacity." after the "Permanent rooms aren't deleted and can have passwords/custom URLs."